Backend/starter_project_zsilentcoders/BlogApp3/BlogApp.Application/Features/Review/DTOs/Validators/UpdateIsResolvedValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using BlogApp.Application.Contracts.Persistence;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Application.Features.Review.DTOs.Validators
+{
+    public class UpdateIsResolvedValidator: AbstractValidator<ReviewIsApprovedDto>
+    {
+        public UpdateIsResolvedValidator(IUnitOfWork unitOfWork)
+        {
+            RuleFor(p => p.IsApproved).NotNull().WithMessage("{PropertyName} must be present");
```

Comment:
You could add a validation here to check if the review exists(or you should check it in your handler-->while updating the isApproved attribute)